scripts/2017/interactive-bots.py

```diff
@@ -0,0 +1,117 @@
+import argparse
+from upload import upload_task
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-f', '--force', dest='force', action="store_true", default=False)
+args = parser.parse_args()
+
+bots_to_create = [
+    ('Chess', ('lets you play chess against the computer or other people on Zulip')),
```

Comment:
I guess for this, we should make really sure that the student uses an existing chess engine library ; otherwise it'll be way too much work.  I think we should somehow put that into the description.